docs/source/install.rst

```diff
@@ -64,7 +64,7 @@ Test Dependencies
 
     conda install -n ucx -c rapidsai -c nvidia -c conda-forge \
         pytest pytest-asyncio \
-        cupy "numba>=0.46" rmm \
+        cupy "numba>=0.57" rmm \
         distributed
```

Comment:
This list of testing dependencies doesn't align with the contents here: https://github.com/rapidsai/ucx-py/blob/6f7c1aec5889eb08e488a3c9e85cdf5ad25e1368/dependencies.yaml#L73-L81

Differences are in rmm/cudf and version pinnings for numba.

Reply:
Good point. This should now be fixed. Please let me know what you think :)